openadapt/window/_windows.py

```diff
@@ -1,11 +1,63 @@
 from pprint import pprint
+import pygetwindow as gw
 import pickle
 import time
 
 from loguru import logger
 import pywinauto
 
 
+def get_active_window_state_pgw(read_window_data: bool) -> dict:
+    try:
+        active_window = gw.getActiveWindow()
+        if not active_window:
+            raise RuntimeError("No active window found.")
+    except RuntimeError as e:
+        logger.warning(e)
+        return {}
+
+    meta = {
+        "title": active_window.title,
+        "left": active_window.left,
+        "top": active_window.top,
+        "width": active_window.width,
+        "height": active_window.height,
+        "rectangle": {
```

Comment:
This data structure should exactly match the existing one. What do you think about removing the `rectangle` property?